larndsim/consts.py

```diff
@@ -186,3 +195,11 @@ def load_detector_properties(detprop_file, pixel_file):
     n_pixels = len(np.unique(xs))*ntiles_x, len(np.unique(ys))*ntiles_y
     n_pixels_per_tile = len(np.unique(xs)), len(np.unique(ys))
 
+
+# hacky stuff for lightLUT that probably belongs in the module descriptor files
+# Numpy array containing the number of TPCs on the x,y and z axis respectively
+n_tpc = [2,1,1] 
+# Numpy array showing the x,y and z sizes of the modules of the TPC
+ModuleDimension = [670.,2022.414,670.]
+# Numpy array displaying the number of modules in the x,y and z axis
+n_mod = [1,1,1]
```

Comment:
This `consts.py` module is not the best and needs refactoring, but could we avoid adding more variables and instead store them in the YAML configuration file for the detector? I don't think you need to store the number of TPCs or module on each axis explicitly, they can be derived from existing information. Also, this is for Module0 but we want to be able to extend this to ND-LAr easily.

Reply:
light sim now gets the voxelization from tpc_borders, so this is removed